src/mailcowAPI.ts

```diff
@@ -1,7 +1,7 @@
 ﻿import * as https from 'https';
 import MailCowClient from 'ts-mailcow-api';
-import { ACLEditRequest, Alias, MailboxEditRequest, MailboxPostRequest } from 'ts-mailcow-api/src/types';
-import { Mailbox, MailboxEditAttributes } from 'ts-mailcow-api/dist/types';
+import { ACLEditRequest, Alias, BaseBccMap, MailboxEditRequest, MailboxPostRequest } from 'ts-mailcow-api/src/types';
+import { GetBccMapResponse, Mailbox, MailboxEditAttributes } from 'ts-mailcow-api/dist/types';
```

Comment:
Thats no good.... you should not have to "from" in the dist or src? Let me fix that hahaha